src/DependencyInjection/Configuration.php

```diff
@@ -54,6 +54,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('revision_field_name')->defaultValue('rev')->end()
                 ->scalarNode('revision_type_field_name')->defaultValue('revtype')->end()
                 ->scalarNode('revision_table_name')->defaultValue('revisions')->end()
+                ->scalarNode('with_foreign_keys')->defaultValue('default')->end()
```

Comment:
The default value should be a boolean and not `default` no ?

Reply:
Yes, I fixed it.

Reply:
And renamed the parameter name to disable_foreign_keys